extensions/wikia/SpecialDiscussions/templates/SpecialDiscussions_inputForm.mustache

```diff
@@ -0,0 +1,5 @@
+<p>{{discussionsInactiveMessage}}</p>
+<form method="post" class="form-discussions">
+    <input type="hidden" name="editToken" value="{{editToken}}">
+	<input type="submit" value="{{activateDiscussions}}"/>
```

Comment:
Minor; indented too far.  Probably was done by PHPStorm because the `input` on the line above isn't terminated with a `/>`